proto/buf/alpha/registry/v1alpha1/resource.proto

```diff
@@ -0,0 +1,45 @@
+// Copyright 2020-2022 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.registry.v1alpha1;
+
+import "buf/alpha/registry/v1alpha1/plugin.proto";
+import "buf/alpha/registry/v1alpha1/repository.proto";
+
+message Resource {
+  oneof resource {
+    Repository repository = 1;
+    Plugin plugin = 2;
+  }
+}
+
+// ResourceService is a service that provides RPCs that allow BSR to query resources
```

Comment:
maybe

```suggestion
// ResourceService manages resources.
```

since in the future it might do more?

Reply:
You're right.